d3a_interface/sim_results/kpi.py

```diff
@@ -221,6 +301,11 @@ def _kpi_ratio_to_percentage(self, area_name):
     def update(self, area_dict, core_stats, current_market_slot):
         if not self._has_update_parameters(area_dict, core_stats, current_market_slot):
             return
+
+        parent_fee = self.area_uuid_to_cum_fee_path.get(area_dict.get('parent_uuid', ''), 0.)
+        area_fee = core_stats.get(area_dict.get('uuid', ''), {}).get('const_fee_rate', 0.)
+        self.area_uuid_to_cum_fee_path[area_dict['uuid']] = parent_fee + area_fee
+
```

Comment:
I'm having some issues trying to understand what object is expected from all these chained `.get()` calls. I think it would be good to refactor this to improve clarity.
Also, why is the default value sometimes `''` and some other times `None` (not here, but in the whole PR)?